### PR TITLE
Add Arista eOS documentation

### PR DESCRIPTION
**Pull request by WannesW**

### CODE REVIEW

File: connector/doc/Arista_eOS_Manager.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+### Web Interface
+
+The web interface is only accessible when the client machine has network access to the product.
+
+## How to use
+
+### Communication Settings
+
+In the **Communication Manager**, it is possible to adjust how the data is retrieved. If it is possible to retrieve the data from different connections, it can be configured on the **Type** parameter (e.g. GNMI, HTTP, etc.).
```
Comment (MariekeGO) [inline connector/doc/Arista_eOS_Manager.md:69]: Do you mean "On the **Communication Manager** page ..."?
Also, what do you mean with "If it is possible to retrieve the data from different connections, it can be configured on the **Type** parameter"?

File: connector/doc/Arista_eOS_Manager.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+When using GNMI it is possible to switch to Subscribe as a **Method**. When using an **Interval** of 0 or On Change the device will push the value of a parameter when it changes. When provided it will report back in the given interval.
```
Comment (MariekeGO) [inline connector/doc/Arista_eOS_Manager.md:71]: Could you clarify this so it's more clear from the perspective of a user? 
- Why are "Subscribe" and "Method" capitalized? Are these things the user can see in the UI? If no, there's no reason for capitalization here, because this will only cause confusion. If yes, could you make it more clear what the user can actually do in the UI?
- "When using an **Interval** of 0 or On Change " => is this about the configuration within the connector rather than something a user can configure? If yes, could you rephrase this, so users don't get the impression that they can configure an interval or "On Change"?

File: connector/doc/Arista_eOS_Manager.md (hunk 1 above)
Comment (WannesW) [reply connector/doc/Arista_eOS_Manager.md:69]: Yes I meant "On the Communication Manager page ..." I already changed it on the branch. 
It will be possible to retrieve e.g. the interface table through SNMP or GNMI. The user will able to configure this in the Type Parameter.

File: connector/doc/Arista_eOS_Manager.md (hunk 2 above)
Comment (WannesW) [reply connector/doc/Arista_eOS_Manager.md:71]: I've rewritten this part to be a bit more clear. the capitalized words correnspond with parameters names in the table/page.

**Review by MariekeGO — APPROVED** [review]
